src/cow-react/modules/limitOrders/containers/LimitOrdersReceiptModal/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReceiptModal as ReceiptModalPure } from '@cow/modules/limitOrders/pure/ReceiptModal'
+import { useAtomValue } from 'jotai/utils'
+import { receiptAtom } from '@cow/modules/limitOrders/state/limitOrdersReceiptAtom'
+import { useCloseReceiptModal } from './hooks'
+import { useWeb3React } from '@web3-react/core'
+import { supportedChainId } from '@src/custom/utils/supportedChainId'
+import { CurrencyAmount, Fraction, Token } from '@uniswap/sdk-core'
+import JSBI from 'jsbi'
+
+function calculatePrice({
+  buyAmount,
+  sellAmount,
+  inputToken,
+  outputToken,
+}: {
+  buyAmount: JSBI | undefined
+  sellAmount: JSBI | undefined
+  inputToken: Token
+  outputToken: Token
+}) {
+  const isZero = (x: JSBI) => JSBI.equal(x, JSBI.BigInt(0))
+
+  if (!buyAmount || !sellAmount || !inputToken || !outputToken || isZero(buyAmount) || isZero(sellAmount)) {
+    return null
+  }
+
+  const adjustedBuy = adjustDecimals(buyAmount, inputToken.decimals)
+  const adjustedSell = adjustDecimals(sellAmount, outputToken.decimals)
+  return new Fraction(adjustedBuy, adjustedSell)
+}
+
+const adjustDecimals = (amount: string | JSBI, decimals: number) => {
+  return JSBI.multiply(JSBI.BigInt(amount), JSBI.BigInt(10 ** decimals))
+}
+
+export function LimitOrdersReceiptModal() {
```

Comment:
<img width="1231" alt="image" src="https://user-images.githubusercontent.com/7122625/203937281-7259b14f-861f-4459-84c5-0172e1cefe85.png">

Can't scroll to the bottom when the window height is less than 600px